mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.VideoCompositor.Mixfile do
   use Mix.Project
 
-  @version "0.3.0"
+  @version "0.3.1"
```

Comment:
Are we sure we want to bump the version?

Reply:
This was merged from the master